### ISSUE TITLE
[ios] Telemetry button in attribution UIAlertController doesn't present telemetry UIAlertController from modal view controller

### ISSUE THREAD

**Issue by kmorey** [mapbox/mapbox-gl-native#8980]
**Platform:**

iOS

**Mapbox SDK version:**

3.5.4 via cocoapods

**Steps to trigger behavior**

1. Add modally presented VC
2. Add map view to modally presented VC
3. Run app, navigate to map view
4. Tap attribution button
5. Tap Mapbox Telemetry button

**Expected behavior**

Telemetry UIAlertController displayed

**Actual behavior**

No change in UI (other than button highlight state). Error shown in console:

```
Warning: Attempt to present <UIAlertController: 0x7ff1c060e360> on <MapboxModalAttribution.ViewController: 0x7ff1c280bb40> whose view is not in the window hierarchy!
```

**Demo Project**

Thanks to @cmoulton from #8782 for the demo project that exhibits this behavior as well!

https://github.com/cmoulton/MapboxModalAttributionTest

**Comment by jmkiley** [comment]
@incanus and I took a look at this. This may be because the method to check for the topmost `UIViewController` (implemented in #8837) is not called on the Telemetry `UIAlertController`. Opened https://github.com/mapbox/mapbox-gl-native/pull/9027 to address this.

**Comment by jmkiley** [comment]
Fixed in #9027
